docs/new_source.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: page
+title: New Data Source
+permalink: /data/new
+nav_order: 1
+has_children: true
+parent: Data sources
+---
+
+# Contributing A New Data Source
+
+Data contributions are welcome. OSV.dev readily accepts security related records from open source projects, given they are provided in the [OSV format](https://ossf.github.io/osv-schema/). Publishing records to OSV is straightforward: reserve a prefix in the OSV-Schema, and then prepare and publish your data via a Git repository or other supported method. The step by step instructions are as follows:
+
+- [ ] Open an [issue](https://github.com/google/osv.dev/issues) using the 'new data source' template.  
+        
+- [ ] Prepare your data \- refer to the [OSV Schema](https://ossf.github.io/osv-schema/) documentation for information on how to properly format the data so it can be accepted.  
+        
+- [ ] Create a PR to [reserve a prefix in the OSV-Schema](https://ossf.github.io/osv-schema/#id-modified-fields) ([example](https://github.com/ossf/osv-schema/pull/219)). We review the records you start publishing for OSV Schema [correctness](https://github.com/ossf/osv-schema/tree/main/validation) and [quality](https://google.github.io/osv.dev/data_quality.html) as part of reviewing and merging this PR.
+
+- [ ] Prepare and publish your records via a public Git repository ([example](https://github.com/AlmaLinux/osv-database/tree/master)). If this method isn’t ideal, we also support publishing records from GCS bucket ([example](https://storage.googleapis.com/android-osv/)) or [REST endpoint](/data/new/rest-api).  
+        
+- [ ] To support API querying, please create a PR to extend [purl\_helpers.py](https://github.com/google/osv.dev/blob/master/osv/purl_helpers.py) and create a new ecosystem in [\_ecosystems.py](https://github.com/google/osv.dev/blob/master/osv/ecosystems/_ecosystems.py). You can refer to existing examples showing how to implement support for [Semver](https://github.com/google/osv.dev/blob/139de7b69a2ea39e2113309b3a0a47aab920ddcf/osv/ecosystems/_ecosystems.py#L45) and [non-Semver](https://github.com/google/osv.dev/pull/3430) ecosystems.  
```

Comment:
nit: Not sure how important it is to call out, but a couple of these steps aren't needed if adding a data source for an existing ecosystem.